gateway/platforms/vertx3/vertx3/src/main/java/io/apiman/gateway/platforms/vertx3/api/auth/KeycloakOAuthFactory.java

```diff
@@ -143,8 +148,42 @@ public AuthHandler addAuthorities(Set<String> authorities) {
         };
     }
 
+    private static HttpClientOptions getHttpClientOptionsForKeycloak(JsonObject config) {
+        HttpClientOptions options = new HttpClientOptions();
+
+        if (config.containsKey("ssl-required")) {
+            if (!config.getString("ssl-required").toLowerCase().equals("none")) {
+                options.setSsl(true);
+            }
+        }
+
+        if (config.containsKey("allow-any-hostname")) {
+            Object value = config.getValue("allow-any-hostname");
+            if (value.equals(true) || value.equals("true")) {
+                options.setVerifyHost(false);
+            }
+        }
+
+        if (config.containsKey("disable-trust-manager")) {
+            Object value = config.getValue("disable-trust-manager");
+            if (value.equals(true) || value.equals("true")) {
```

Comment:
With `if (value.equals(true) || value.equals("true")) ` you are able to use System Properties in the conf-X.json like so:

```
"disable-trust-manager": "${devMode}",
"allow-any-hostname" : "${allowAnyHost}"
```